includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -1367,12 +1386,7 @@ private function process_stylesheet( $stylesheet, $options = [] ) {
 		);
 
 		$cache_key = md5( $stylesheet . wp_json_encode( $cache_impacting_options ) );
-
-		if ( wp_using_ext_object_cache() ) {
-			$parsed = wp_cache_get( $cache_key, $cache_group );
-		} else {
-			$parsed = get_transient( $cache_group . '-' . $cache_key );
-		}
+		$parsed    = $this->cache_pool->get( $cache_key );
```

Comment:
This looks to work well.

Once the stylesheet is stored in a transient. this `->get()` method retrieves it.

<img width="1290" alt="sanitizer-here" src="https://user-images.githubusercontent.com/4063887/65277577-d198c600-daef-11e9-9457-d76e37060333.png">